extensions/widgets/clock/clock.lcb

```diff
@@ -76,6 +80,188 @@ UTC that the clock should display.
 property timeZone		get mTimeZone		set setTimeZone
 metadata timeZone.editor is "com.livecode.pi.timezone"
 metadata timeZone.label is "Timezone"
+
+/**
+Syntax: set the dayNumberColor of <widget> to <color>
+Syntax: get the dayNumberColor of <widget>
+
+Summary: Determines the color of the numbers on the clock during the day
+
+Description:
+Use the <dayNumberColor> property to get or set the text color of the numbers
+on the face of the clock during the day. To determine if the clock is
+displaying its daytime or night-time colors use the read-only property
+<isDay>.
+*/
+property dayNumberColor get getDayNumberColor set setDayNumberColor
+metadata dayNumberColor.editor is "com.livecode.pi.color"
+metadata dayNumberColor.default is "0,0,0"
+metadata dayNumberColor.section is "Colors"
+metadata dayNumberColor.label is "Number color (day)"
```

Comment:
There's (probably) no reason not to allow the colors to have alpha values - all you'd need to do is use the 
`"com.livecode.pi.colorwithalpha"` editor and make sure all the default color strings have full opacity as the fourth item

Reply:
I borrowed that from the colorSwatch widget, so I didn't realise that there was a dedicated PI editor for alpha colors. Agree it's probably best - by default the dayFaceColor is partially transparent.